### DISCUSSION TITLE
403 error message

### DISCUSSION THREAD

**Discussion by OpenK54** [ZoeyVid/NPMplus#315] [Q&A] [answered]
I have several servers behinde the proxy. I can access them from the web and browse the servers gui. But it is not possible to access any file. Debbuging the browser I see 403 forbidden responses in the browser, but these are not present in the servers log. I'm happy to get any hints as to what the issue could be.

**Comment by Zoey2936** [comment]
do you have modsec and corerulset enabled?

**Reply by OpenK54** [reply]
Yes, both.

**Reply by Zoey2936** [reply, marked as answer]
then thats the problem, please try with corerulset disabled

**Reply by OpenK54** [reply]
You nailed it! This is the problem. What does this setting do exactly?

**Reply by Zoey2936** [reply]
It enabled this security settings: https://github.com/coreruleset/coreruleset

**Reply by Zoey2936** [reply]
they sometimes overblock

**Reply by OpenK54** [reply]
Thanks for the quick response and troubleshooting!
